### PR TITLE
Adds internship simplified challenge

### PR DESCRIPTION
**Pull request by klizhentas**

### CODE REVIEW

File: challenges/systems-intern/worker-library.tex [hunk 1]
```diff
@@ -0,0 +1,205 @@
...
+
+This is the question we expect candidates to figure out on their own:
+
+``What version of Go should I use? What dependency manager should I use?''
+
+Unless specified in the requirements, pick the solution that works best for you.
+
+\section{Tools}
+
+This task should be implemented in Go, C++, Rust or Java and should work on 64-bit Linux machines.
```
Comment (r0mant) [inline challenges/systems-intern/worker-library.tex:189]: Didn't we remove Java from our allowed languages, or is that ok specifically for the interns?

**Review by r0mant — APPROVED** [review]

**Review by russjones — APPROVED** [review]
Looks fine to me, but maybe we can just include `L0` to the current challenge with the worker library because the rest of the challenge appears to be the same? Something like this:

----

### L0

#### Library

Worker library with methods to start/stop/query status and get an output of a running job.

### L1

#### API

Add HTTPS API to start/stop/get status of a running process. Use basic authentication.

#### Client

Client command should be able to connect to worker service and schedule several jobs. Client should be able to query result of the job execution and fetch the logs

File: challenges/systems-intern/worker-library.tex (hunk 1 above)
Comment (klizhentas) [reply challenges/systems-intern/worker-library.tex:189]: That's specifically for internships

**Comment by klizhentas** [comment]
@russjones it's slightly different because we have to have different offer procedure for internships as well (university requirements)

**Comment by russjones** [comment]
@klizhentas Alright :+1:.

**Review by knisbet — APPROVED** [review]

**Pull request merged**